### ISSUE TITLE
Clicking a Text inside a button trigger both callbacks

### ISSUE THREAD

**Issue by yairopro** [necolas/react-native-web#1343]
**The problem**
Clicking a `Text` inside a button (`TouchableOpacity`) trigger both `onPress` callbacks.

**How to reproduce**
Put a `Text` with a `onPress` callback inside a `TouchableOpacity`.
Demo (RNW): https://codesandbox.io/s/n4o9o96m0m
Demo: (RN): https://snack.expo.io/SJmyp1mhN
(See logs in demos)

**Expected behavior**
The same behavior as on RN: Only the `Text`'s `onPress` callback is called.

**Environment (include versions)**

* React Native for Web (version): ^0.11.2
* React (version): ^16.8.5
* Browser: Chrome 74

**Comment by kasadawa** [comment]
Hi ,
lets look at the example below:
``` 
<div onClick={()=>console.log('parent')}>
      Some parent text
     <p onClick={()=>console.log('child')> This is the child text</p>
</div>
```
So the expected behaviour when you click on the <p> text is to call both onClick events. This is caused because the event it bubbling up from the child -> parent. 

I think triggering the both onClick events when you click  on the child <p> is the correct behaviour. Then if you want you can stop the propagation with `e.stopPropagation()`

But still you are right that you should expect the same behaviour.

**Comment by yairopro** [comment]
Hi @kasadawa 

I know bubbling is the way events work on the web.
But since RNW makes web development be like an application development, and its purpose is to allow the use of RN on the web platform seamlessly, then it should work the same way. In application development, correct if you think I am wrong, events don't bubble.

Furthermore, if you replace the link with another button, it works fine. The event doesn't bubble.
Demo: https://codesandbox.io/s/pk3k23w4y0

PS: I admire @necolas ' work on this project, and I'm not criticizing it in any way.

**Comment by necolas** [comment]
This was fixed in 0.13
